bangazonapi/models/user.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
```

Comment:
I don't think we need this User model anymore. I would suggest deleting this user.py file so we don't accidentally import the wrong User sometime in the future.